### DISCUSSION TITLE
doorbell_graham_cracker Model Name changeable?

### DISCUSSION THREAD

**Discussion by unstressable** [dgreif/ring#1152] [General]
Hi,

is it possible to change the Model Name "doorbell_graham_cracker" in HomeApp?

Looks a bit unprofessional :D

**Comment by smockle** [comment]
Given that this has shown up in the Ring app itself (https://community.ring.com/t/video-doorbell-wired-unusual-device-name-hacked/50292), it looks like an issue on Ring’s side.

**Reply by unstressable** [reply]
Hmm thanks!
Known since 2021 and still there :(

isnt it possible to override this setting in homebridge?

**Reply by dgreif** [reply]
I agree using candy for device types isn't the most professional, but it's something Ring has been doing for a lot of their code behind the scenes lately 🤷 . That said, `doorbell_graham_cracker` is truly the technical "device type" for these cameras. We do already have [a mapping](https://github.com/dgreif/ring/blob/bdb72b56a6446cb957f9583db0487c1c35275a43/packages/ring-client-api/ring-types.ts#L114) that should cause this to show up as a `Doorbell Wired`. In HomeKit, it should say `Doorbell Wired (doorbell_graham_cracker)` under the `Model` field. If it's missing the `Doorbell Wired`, let me know and I can look into it more.

**Reply by unstressable** [reply]
It looks like this:
![9E8A8DA2-30EF-49C7-9E93-09C355625415](https://user-images.githubusercontent.com/50423935/218187431-745723c0-7430-47eb-b517-c566ab06d61d.jpeg)

i just want to see „Doorbell Wired“ but not the Cracker part.
Is it possible to change?

**Reply by dgreif** [reply]
I could simply remove the technical name in `()` from that area, but I think that's valuable information to have for debugging purposes. I don't think there is anything else to be done here, sorry!

**Reply by unstressable** [reply]
ok, no worries.
I you have time, you could give us an override feature to set a free configurable name.
